cmd/generate/config/rules/grafana.go

```diff
@@ -0,0 +1,65 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func GrafanaApiKey() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Grafana api key (or Grafana cloud api key)",
+		RuleID:      "grafana-api-key",
+		SecretGroup: 1,
+		Regex:       generateUniqueTokenRegex(`eyJrIjoi[A-Za-z0-9]{70,}={0,2}`),
+		Keywords:    []string{"eyJrIjoi"},
+	}
+
+	// validate
+	tps := []string{
+		generateSampleSecret("grafana-api-key",
+			"eyJrIjoi"+
+				secrets.NewSecret(alphaNumeric("70"))),
+	}
+	return validate(r, tps, nil)
+}
+
+func GrafanaCloudApiToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Grafana cloud api token",
+		RuleID:      "grafana-cloud-api-token",
+		SecretGroup: 1,
+		Regex:       generateUniqueTokenRegex(`glc_[A-Za-z0-9+/]{32,}={0,2}`),
+		Keywords:    []string{"glc_"},
```

Comment:
same goes for here, could `{32,}` have an upper bound?

Reply:
Would [400](https://github.com/zricethezav/gitleaks/pull/959#discussion_r950198461) be a reasonable upper bound for you ?